docs/rfcs/014-born_digital_workflow/README.md

```diff
@@ -0,0 +1,176 @@
+
+| Step                 | Description |
+|----------------------|-------------|
+| Disk level appraisal | High level appraisal to determine what storage media is in scope. |
+| File level appraisal | Selection of in scope files from selected storage media. |
+| Sensitivity check    | Policy-led review of selected files to determine whether they contain sensitive material. |
+
+### Cataloguing
+
+Once an initial appraisal has been completed, the files in scope for archiving need to be arranged into archive items and catalogued in Calm. Items should be created in Calm to represent each related set of files, which will be submitted and stored as a single package in the storage service.
+
+This means that an item in Calm corresponds to a bag in the storage service, not an individual file. File-level description is fully automated and the metadata generated is stored in a METS file created by the Archivematica processing workflow.
+
+| Step                   | Description |
+|------------------------|-------------|
+| Find/create collection | Find, or create if necessary, the collection in Calm that the items will be part of. |
+| Find/create series     | Find, or create if necessary, the series and sub-series within the collection that the items will be part of. |
+| Create items           | For each set of related files, create an item in Calm that represents and describes them collectively in the catalogue. This should describe the files as a set, not list details of individual files. |
+
+### Transfer
+
+Once each set of related files as been identified, they need to be assembled into a package that can be transferred into Archivematica for processing and storage. This package is a simple zip file, with the addition of a minimal set of descriptive metadata.
+
+This metadata is used for identification only. The canonical version of the metadata, which is also the data that will be displayed to end users, remains the data in Calm. Similarly, whilst Archivematica does have support for rights information, we will not be supplying this through the workflow. Calm will be the sole location for rights information.
+
+The package created for transfer into Archivematica should reflect the original order of the files, as this structure will be maintained in the storage service and in the file-level metadata presented to users.
+
+The zip file should have the following structure:
+
+	PPABC_1234.zip
+	|-- file1.txt
+	|-- file2.doc
+	|-- file3.jpg
+	|-- folder1
+	    \-- [...] 
+	|-- folder2
+	    \-- [...] 
+	|-- [...]
+	\-- metadata
+	    \-- metadata.csv
+
+ Note the addition of a `metadata` directory, with a single file named `metadata.csv`, which contains item level metadata from Calm. `metadata.csv` should have the following structure:
+
+	filename,dc.title,dc.creator,dc.date,dc.identifier
+	objects,<Title>,<CreatorName>,<Date>,<RefNo>
```

Comment:
If this is viewed as inventory rather than cataloguing, it makes more sense to align the metadata to the Calm accession record and the SPECTRUM inventory standard (the minimal viable record). This would require:

- Object (item number) = <RefNo> plus <AccNo> and possibly <AccNo> sub-number for digital components
- Name = <Title>
- Number of objects = <Extent> Do we currently record this in accession records, Jenny or does this require some amendments to the Calm accessions template?
- Brief description = <Title>
- Current location - can presumably be added after processing through in S3?
- Current owner = <Repository>
- Recorder and recording date for this information - again, added via Archivematica workflow?

Reply:
This of course assumes that 'metadata.csv' can be customised? 

We can't supply <CreatorName> from either Accession or Catalogue item records.
Nor is <RefNo> on its own sufficient to identify a single accession.